### PR TITLE
GHA Action Question & template fix

### PR DESCRIPTION
**Pull request by riosengineer**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.`
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [x] Documentation changes
- [ ] Other

## What's new?
Added new question in GitHub Actions (Question 076) using https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations as reference / source.

Removed trailing ` from the CONTRIBUTING.md file

Removed trailing ` from PULL_REQUEST_TEMPLATE.md file

### CODE REVIEW

**Comment by FidelusAleksander** [comment]
Hey I see You are still doing updates so I will review it later today.

Btw there are hugo commands in the contributing guide that will automatically create new files for you with correct structure.

Commands: https://github.com/FidelusAleksander/ghcertified/blob/master/CONTRIBUTING.md#adding-new-question
Structure: https://github.com/FidelusAleksander/ghcertified/blob/master/archetypes/questions.md?plain=1

File: content/questions/actions/question-076.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "How many reusable workflows can you call from a single workflow file? "
```
Comment (FidelusAleksander) [inline content/questions/actions/question-076.md:3]: Let's rephrase this to say "maximum" because right now all answers are correct

```suggestion
title: "What's the maximum amount of reusable workflows can you call from a single workflow file? "
```

File: content/questions/actions/question-076.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+question: "Question 076"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations
+1. [] 0
+1. [] 1
+1. [x] 20
+1. [] 10
```
Comment (FidelusAleksander) [inline content/questions/actions/question-076.md:13]: Need's spaces in empty brackets and I've made it a standard to have the first answers to be correct. Doesn't change anything but keeps the repo consistent

```suggestion
1. [x] 20
1. [ ] 5
1. [ ] 1
1. [ ] 10
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Small fixes and it's good to go!

**Comment by riosengineer** [comment]
Thanks @FidelusAleksander 🙂

File: content/questions/actions/question-076.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "What's the maximum amount of reusable workflows can you call from a single workflow file? "
+question: "Question 076"
```
Comment (FidelusAleksander) [inline content/questions/actions/question-076.md:4]:
```suggestion
archetype: "questions"
question: "What's the maximum amount of reusable workflows can you call from a single workflow file? "
title: "Question 076"
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
It's good that I check the preview site before merging using Codespaces :D 

image

**Comment by riosengineer** [comment]
@FidelusAleksander  nice spot. Apologies. I did use the hugo commands but must've pasted something from another question as a reference point! Oops. I'm new to contributing to projects 🙂

**Review by FidelusAleksander — APPROVED** [review]
All good @riosengineer 👍 

Thanks for contributing and good luck on your exam ! 🚀
